src/main/java/com/team4/testingsystem/exceptions/FileNotFoundException.java

```diff
@@ -3,6 +3,6 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
 
-@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "Not found")
-public class NotFoundException extends RuntimeException{
```

Comment:
What do you think about NotFoundException and its extensions: FileNotFound, QuestionNotFound etc?